python/ray/data/context.py

```diff
@@ -247,6 +249,8 @@ class DataContext:
         log_internal_stack_trace_to_stdout: Whether to include internal Ray Data/Ray
             Core code stack frames when logging to stdout. The full stack trace is
             always written to the Ray Data log file.
+        raise_original_exception: Whether to raise the original exception encountered
```

Comment:
since this is only for map operators, can we include it in the context attribute name?
```suggestion
        raise_original_map_exception: Whether to raise the original exception encountered in map UDF
```

Reply:
and also update the DEFAULT variables used in this file accordingly.

Reply:
fixed!